source/_components/switch.rainbird.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Rain Bird Switch"
+description: "Instructions on how to integrate your Rain Bird LNK WiFi Module as Switches within Home Assistant."
+date: 2017-08-25 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rainbird.png
+ha_category: Hub
+ha_release: 0.50
+ha_iot_class: "Local Polling"
+---
+
+This `rainbird` switch platform allows interacting with [LNK WiFi](http://www.rainbird.com/landscape/products/controllers/LNK-WiFi.htm) module of the Rain Bird Irrigation system in Home Assistant.
+
+To enable stations as switches inside Home Assistant, add the following to your `configuration.yaml` file:
+
+```yaml
+switch:
+    platform: rainbird
+    sprinkler_1:
+      zone: 1
+      friendly_name: "Front sprinklers"
+      trigger_time: 20
+      scan_interval: 10
+    sprinkler_2:
+      friendly_name: "Back sprinklers"
+      zone: 2
+      trigger_time: 20
+      scan_interval: 10
+```
+
+Configuration variables:
+
+- **zone** (*Required*): Station zone identifier. Wich zone / valve to start.

```

Comment:
**Spelling & Grammar**:
- `Wich` -> `Which`
- `zone / valve` -> `zone/valve` (remove extra spaces)

Secondly, I found the sentence to be a little strange. `Which zone/value to start.` is it just the starting? Also is there an action involved here? Or is it just the zone identifier? I think it would be appropriate to remove that sentence altogether, but I'm not sure. What do you think?